system-int-test/test/cmr/system_int_test/search/granule_orbit_number_search_test.clj

```diff
@@ -19,65 +20,65 @@
         gran5 (d/ingest "PROV1"
                         (dg/granule-with-umm-spec-collection coll1 (:concept-id coll1)
-                                    {:orbit-calculated-spatial-domains [{:start-orbit-number 2.7
-                                                                         :stop-orbit-number 3.5
+                                    {:orbit-calculated-spatial-domains [{:start-orbit-number 3
+                                                                         :stop-orbit-number 4 
                                                                          :equator-crossing-longitude 0
                                                                          :equator-crossing-date-time "2011-02-01T12:00:00Z"}]}))
         gran6 (d/ingest "PROV1"
                         (dg/granule-with-umm-spec-collection coll1 (:concept-id coll1)
-                                    {:orbit-calculated-spatial-domains [{:start-orbit-number 4.0
-                                                                         :stop-orbit-number 5.0
+                                    {:orbit-calculated-spatial-domains [{:start-orbit-number 4
+                                                                         :stop-orbit-number 5
                                                                          :equator-crossing-date-time "2011-02-01T12:00:00Z"}]}))
         gran7 (d/ingest "PROV1"
                         (dg/granule-with-umm-spec-collection coll1 (:concept-id coll1)
                                     {:orbit-calculated-spatial-domains [{:orbital-model-name "OrbitalModelName"
-                                                                         :start-orbit-number 7.0
-                                                                         :stop-orbit-number 10.0
+                                                                         :start-orbit-number 7
+                                                                         :stop-orbit-number 10
                                                                          :equator-crossing-longitude 0
                                                                          :equator-crossing-date-time "2011-02-01T12:00:00Z"}]}))]
     (index/wait-until-indexed)
 
     (testing "search by exact orbit number"
-      (are [items orbit-range]
-           (d/refs-match? items (search/find-refs :granule {:orbit-number orbit-range}))
+      (are3 [items orbit-range]
+            (d/refs-match? items (search/find-refs :granule {:orbit-number orbit-range}))
 
-           ;; search by exact orbit number
+           "search by exact orbit number"
            [gran1 gran2 gran3] "1"
-           ;; search by orbit number range
+           "search by orbit number range"
            [gran1, gran2, gran3, gran4] "1,2"
-           ;; search by orbit number range with rational numbers
```

Comment:
Now I'm really unsure whether orbit numbers are integers. It seems like we purposely allowed rational numbers.

Reply:
Clarified with Jon Pals.

> Ever since I started working on ECS, orbit numbers have been integers.  It doesn't make sense to me to say that a satellite is on orbit 4.6.  I think that non-integer orbit numbers should be rejected.

Reply:
Yeah. this is what I expected. We will only allow integers for start/stop orbit numbers.

Reply:
Man, I _love_ getting information back like that: people who really know their stuff giving us the knowledge to make clear technical decisions. It's a dream for engineers :-)